lib/node-arguments.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const trim = require('lodash/trim');
+
+// Parse formats:
+// --param=--parameter --abc
+//         |_this____|
+// --param=" --arg1 --arg2 param " --abc
+//         |_this part___________|
+function parseCliParameter(str) {
+	str = str[0].startsWith('"') && str.endsWith('"') ? str.slice(1, str.length - 1) : str;
+	return trim(str)
+		.split(' ');
+}
+
+function normalizeNodeArguments(nodeArguments) {
+	const parsed = Array.isArray(nodeArguments) ? nodeArguments : parseCliParameter(nodeArguments);
+
+	const detectedInspect = parsed.find(arg => /^--inspect(-brk)?(=|$)/.test(arg));
+
+	if (detectedInspect && detectedInspect.includes('=')) {
+		throw new Error('The \'nodeArguments\' configuration must not contain inspect with port.');
+	}
+
+	const mainProcessArgs = process.execArgv.filter(arg => !detectedInspect || !/^--inspect(-brk)?(=|$)/.test(arg));
```

Comment:
I'm not sure it's such a problem if these flags are provided twice, or with a port.

Reply:
This is problem, because ava run multiple workers, and they cannot use same port

Reply:
Oh I see, fair point. My hunch is to not correct for this, for now. We should have a better solution for debugging regardless.

Reply:
If this not needed, we can not parse args, and just concatenate from process.execArgv and config

Reply:
Exactly, much easier!

Although we also have a way of providing the flags to the AVA CLI. So we still need some parsing I suppose.